cryostatio/cryostat-web
# feat(dashboard): add JVM Details card

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #894
Fixes: #895
Depends on https://github.com/cryostatio/cryostat-web/pull/891

## Description of the change:
This adds a simple dashboard card that simply displays basic information about the selected target JVM such as its connection URL, alias, and labels/annotations.

## Motivation for the change:
This provides users with a nicer presentation of the data currently displayed on the `<TargetSelect/>` component expansion, which the user might need to know in order to define matchExpression credentials, automated rules, or to filter/sort through the All Archives table. A follow-up PR will add further MBean-based information to this card as well.

## How to manually test:
1. *Run CRYOSTAT_IMAGE=quay.io... sh smoketest.sh...*
2. Select a target JVM
3. Add the JVM Details dashboard card
4. Check that the details are updated with different target selections and that the details match the target

![image](https://user-images.githubusercontent.com/3787464/221054066-ccb00589-d3ae-4db0-90a4-fad4c4a1693c.png)
